packages/pf-codemods/index.js

```diff
@@ -73,10 +74,11 @@ function main() {
       }
     }
 
+    currentOptions.configFile = undefined;
```

Comment:
Can we just do away with the config file entirely on lines 87-89? There's not that much config actually in it...

Reply:
Yes we can, I can change it here, but I have it in the --exclude switch changes I am making.